analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseUnixEpoch.Framework.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Globalization;
+
+public class Program
+{
+    private readonly DateTime Epoch = new DateTime(1970, 1, 1); // Compliant
```

Comment:
`// Compliant - the UnixEpoch property is only available in .NET Core 2.1/.NET Standard 2.1 and newer, the .NET Framework does not support it`